salsa_dancing_molecules/capacity_NVE.py

```diff
@@ -0,0 +1,46 @@
+"""
+Calculates the specific heat capacity for an NVE ensamble.
+
+Takes an atoms object
+Returns the capacity as a float
+
+"""
+
+import numpy as np
+from ase import units
+from ase.io.trajectory import Trajectory
+from .ensemble_energies import (get_kine_sqr_ensemble,
+                                get_kine_ensemble_sqr)
+
+
+def get_NVE_heat_capacity(traj, t0):
+    """Calculat Heat Capacity for a NVE ensamble.
+
+    Input:
+        traj: file   - Trajectory-file
+        t0: int      - Timestep when equilibrium starts
+
+    Output:
+        int - time average heat capacity, time average over equalibrium
+    """
+    config = Trajectory(traj)
+    return calculate_NVE_heat_capacity(config, t0)
+
+
+def calculate_NVE_heat_capacity(config, t0):
+    """Calculate Heat Capacity for NVE snemble.
+
+    Imput:
+        config: readable trajectory file - traj file containing atom objects
+        t0: int                          - Timestep when equilibrium starts
+
+    Output:
+        int - time average heat capacity, time average over equalibrium
+    """
+    atom = config[-1]
+    T2 = atom.get_temperature()**2
+    k_sqr_ensemble = get_kine_sqr_ensemble(config, t0)[-1]
+    k_ensemble_sqr = get_kine_ensemble_sqr(config, t0)[-1]
+    return (len(atom)*units.kB*1.5/(1 - (k_sqr_ensemble
```

Comment:
If I've interpreted this code correctly, we are using an average for the entire equilibrium time period for the energy, but we use the temperature of the last atoms object. Shouldn't the temperature also be an average?

Reply:
True, will change to the average.